### ISSUE TITLE
Mac:  new Export dialog is not properly accessible to keyboard-only users - gets stuck in Channels

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#4997]
### Bug description

Testing on macOS 12.6.7 with latest alpha: audacity-macOS-3.4.0-alpha-20230804+ca2098e-x86_64

Shows that when using the keyboard to Tab through the various entries in the new export dialog:
a) the tabbing gets stuck in the three channels radio buttons circling through them ad infinitum 
b) there appears to be no way to change the setting of the Channels from the default offered on launch if File>Export

This issue was introduced as apart of @vsverchinsky 's new export

### Steps to reproduce

1. Launch latest 3.4.0 alpha:   audacity-macOS-3.4.0-alpha-20230804+ca2098e-x86_64
2. get some audio
3. File > Export
4. Observe: initial focus is on File Name (proper, expected behavior)
5. Press tab several times
6. Observe: tabbing gets stuck in the 3 radio buttons for Channels
7. Try to make a different setting for Channels
8. Observe:  you can't

### Expected behavior

Should be able to tab through the whole set of entries  in the File>Export dialog

Should be able to reset the Channels setting using just the keyboard

### Actual behavior

tabbing gets stuck in the 3 radio buttons for Channels

and you can't change the Channels setting using just the keyboard

### Audacity Version
latest 3.4.0 alpha - audacity-macOS-3.4.0-alpha-20230804+ca2098e-x86_64

### Operating system
macOS 12.6.7 Momterey

### Additional context
flagging: @DavidBailes and @chinakov  and @vsverchinsky 

This is effectively a regression on 3.3.3 where the Export dialog is totally accessible to keyboard-only users

**Comment by petersampsonaudacity** [comment]
See also:   **Windows: New export dialog: initial focus is not the first control** #4960

**Comment by DavidBailes** [comment]
I checked this out on a friends iMac running MacOS 10.15.7
When you tab to one of the channels radio buttons, you can set it using Space. This appears to be the same for all the groups of radio buttons in Audacity, and is probably acceptable, though not ideal.
A user can escape from being trapped in this group of radio buttons by pressing Ctrl + Tab. Not all users may try that.

**Comment by petersampsonaudacity** [comment]
Many thanks for testing that @DavidBailes , much appreciated

I not the on my Mac
a) in the three button group Ta will cycle through all three radio buttons in turn
b) for whichever button has focus, pressing Enter will indeed change the setting to that radio button
c)  and yes indeed Ctrl+Tab will move the user on out of the Channels 3 radio buttons

I experience similar behavior for setting when I Tab down to the Export Range radio buttons - BUT here a simple Tab take me on ti the Edit Metadata - BUT IMPORTANTLY it skips over the **Trim blank space before first clip** so the keyboard-only user cannot select that and change it.

And similar behavior getting locked in a loop with the **Name Files** section  for **Multiple Files** export.


These behaviors on Mac are far from "discoverable" especially the Ctrl modified Tab to escape from button looping - all is much easier and and more "discoverable" on Windows - and Windows can access the "Trim blank space ..." field.

I can (or could) document these odd Mac behaviors - but really this should be improved for VI users (and those who just prefer to use the keyboard) as it is a **_very_** poor UX fir those users.

Tagging @Tantacrul @LWinterberg as a UX issue

**Comment by petersampsonaudacity** [comment]
On Mac _(unlike Windows)_ the same locked-in radio button looping happens in **Tracks Behaviors** preferences.

But curiously this does **_NOT_** happen on Mac with the radio buttons in **Import/Export** preferences

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
